logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/GetNotificationsUseCase.kt

```diff
@@ -32,17 +34,20 @@ interface GetNotificationsUseCase {
 
 /**
  *
+ * @param observeConnectionList ObserveConnectionListUseCase for getting connection requests
  * @param messageRepository MessageRepository for getting Messages that user should be notified about
  * @param userRepository UserRepository for getting SelfUser data, Self userId and OtherUser data (authors of messages)
  * @param conversationRepository ConversationRepository for getting conversations that have messages that user should be notified about
  * @param timeParser TimeParser for getting current time as a formatted String and making some calculation on String TimeStamp
  * @param messageMapper MessageMapper for mapping Message object into LocalNotificationMessage
- * @param publicUserMapper PublicUserMapper for mapping PublicUser object into LocalNotificationMessageAuthor
+ * @param localNotificationMessageMapper LocalNotificationMessageMapper for mapping PublicUser object into LocalNotificationMessageAuthor
  *
  * @return Flow<List<LocalNotificationConversation>> - Flow of Notification List that should be shown to the user.
  * That Flow emits everytime when the list is changed
  */
+@Suppress("LongParameterList")
 class GetNotificationsUseCaseImpl(
+    private val observeConnectionList: ObserveConnectionListUseCase,
```

Comment:
I don't know if there is some agreement here, but should we use use cases in other use cases? 😄

Reply:
we already have such cases, like in `ObserveConversationsAndConnectionsUseCaseImpl` :)